### PR TITLE
Update CA Certs and Fingerprint

### PR DESCRIPTION
**Pull request by khoih-prog**
Update CA Certs and Fingerprint for wss://echo.websocket.org

Please check [**unable to connect to wss with esp32** #101](https://github.com/gilmaimon/ArduinoWebsockets/issues/101) for more info.

### CODE REVIEW

File: examples/SecuredTwoWay-Esp8266-Client/SecuredTwoWay-Esp8266-Client.ino [hunk 1]
```diff
@@ -35,29 +35,6 @@ const char *ntp1 = "time.windows.com";
 const char *ntp2 = "pool.ntp.org";
 time_t now;
 
-// The hardcoded certificate authority for this example.
-// Don't use it on your own apps!!!!!
-const char ca_cert[] PROGMEM = R"EOF(
```
Comment (gilmaimon) [inline examples/SecuredTwoWay-Esp8266-Client/SecuredTwoWay-Esp8266-Client.ino:40]: Why did you replace the cert here with fingerprint? I think I rather have this example as is (even if the certificate is outdated, to show usage of `setTrustAnchors`)

This doesn't work without changing the certificate?

**Review by gilmaimon — CHANGES_REQUESTED** [review]
Looking good mostly, just look at my comment for the two-way example :)

File: examples/SecuredTwoWay-Esp8266-Client/SecuredTwoWay-Esp8266-Client.ino (hunk 1 above)
Comment (khoih-prog) [reply examples/SecuredTwoWay-Esp8266-Client/SecuredTwoWay-Esp8266-Client.ino]: Already restore the original example.

Comment (khoih-prog) [reply examples/SecuredTwoWay-Esp8266-Client/SecuredTwoWay-Esp8266-Client.ino:40]: Just updated the CA Cert for `SecuredTwoWay-Esp8266-Client` example

**Comment by khoih-prog** [comment]
Now all CA Certs / Fingerprint are updated and tested OK on 3 related examples

**Comment by khoih-prog** [comment]
Already changed as requested. Waiting to be merged. Thanks,

**Pull request merged**